src/App.js

```diff
@@ -13,24 +11,18 @@ function App() {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    // display user input unless string is empty, then display default message
+    // Display user input as textOutput. Or display default message as textOutput if user input is empty: textInput === ""
```

Comment:
Thanks for clarifying the ternary below. I wasn't sure why you structured it this way, but now it makes sense you want to display a default message.